src/phases/tpch/denormalized/Q15.yml

```diff
@@ -11,8 +11,7 @@ TPCHDenormalizedQuery15:
   Repeat: 1
   Database: tpch
   Operations:
-  - OperationMetricsName: Query15CreateView
-    OperationName: RunCommand
+  - OperationName: RunCommand
```

Comment:
Just one question (non-blocking)- as I understand it this still gets tracked as DatabaseOperation.4.4 or similar in OperationMetrics even once the name is removed. Is that intended?

Reply:
> Just one question (non-blocking)- as I understand it this still gets tracked as DatabaseOperation.4.4 or similar in OperationMetrics even once the name is removed. Is that intended?

Oh you are correct, so that's not good! I have just moved the problem to a very obscurely named metric. 

I'll try and clarify the situation with genny. I'll open a feature request if it is not possible to exclude a phase metric. Although I'll probably merge / close this as well.

Thanks!